tests/performance/src/test/java/org/mule/Base64EncodingPerformanceTestCase.java

```diff
@@ -0,0 +1,55 @@
+package org.mule;
```

Comment:
missing license header. You should always run a maven build before submitting PR